### PR TITLE
add documentaion about entity providers and refactor catalog docs

### PR DESCRIPTION
**Pull request by punkle**
Preview link: https://deploy-preview-1225--roadie.netlify.app/docs/catalog/ingesting-data-into-your-catalog/

### CODE REVIEW

**Comment by dtuite** [comment]
I feel like this is unfortunately buried as a sidebar item on the catalog page. Is this page (with a different name instead of "Plugins and Integrations") a better bet? https://deploy-preview-1225--roadie.netlify.app/docs/integrations/

**Comment by punkle** [comment]
Do you mean have a heading called "entity providers" or something like that?

**Comment by Xantier** [comment]
The title looks to be different form other pages: 
image

File: content/docs/catalog/ingesting-data-into-your-catalog/azure-devops/index.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+title: Azure DevOps
+publishedDate: '2024-02-20T21:00:00.0Z'
+description: How to manage entities source from Azure DevOps
 
-humanName: Azure DevOps Provider
-logoImage: '../../../assets/logos/azure-devops/Azure-DevOps-logo.png'
-integrationType: Integration
+humanName: Azure DevOps
+logoImage: '../../../../assets/logos/azure-devops/Azure-DevOps-logo.png'
+entityKinds: [resource, component, domain, system, template, api, user, group]
```
Comment (Xantier) [inline content/docs/catalog/ingesting-data-into-your-catalog/azure-devops/index.md:8]: This should have discovery tag also no?

Comment (punkle) [reply content/docs/catalog/ingesting-data-into-your-catalog/azure-devops/index.md:8]: Possibly. Ill check and add it if so.

**Comment by dtuite** [comment]
https://app.shortcut.com/larder/story/18846/review-of-all-the-ways-to-import-into-the-catalog-docs-that-mention-importing

**Pull request closed without merging**